packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -208,6 +208,35 @@ class Neo4jGraphQL {
         });
     }
 
+    public neo4jValidateGraphQLDocument(): { isValid: boolean; validationErrors: string[] } {
+        let isValid = false;
```

Comment:
As a minor thing I would avoid to mutate the state of these variables, like:
```ts
} catch (error) {
    if (error instanceof Error) {
        const validationErrors = error.message.split("\n\n");
        return { isValid: false, validationErrors };
    }
}
return { isValid: true, validationErrors: [] };
```

Reply:
Hmm yes, I see your point. I can make the adjustment.

Reply:
https://github.com/neo4j/graphql/pull/3341/commits/61ecefcb30069eecd0bdd0ce100e0cd0dcbee81d